code/modules/lighting/lighting_corner.dm

```diff
@@ -137,5 +137,5 @@ GLOBAL_LIST_INIT(LIGHTING_CORNER_DIAGONAL, list(NORTHEAST, SOUTHEAST, SOUTHWEST,
 		return QDEL_HINT_LETMELIVE
 
 	stack_trace("Ok, Look, TG, I need you to find whatever fucker decided to call qdel on a fucking lighting corner, then tell him very nicely and politely that he is 100% retarded and needs his head checked. Thanks. Send them my regards by the way.")
-	// Yeah fuck you anyways.
-	return QDEL_HINT_LETMELIVE
```

Comment:
@Cyberboss I was 100% aware when I wrote that. That's why there's the above `stack_trace`